specs/pvs/corla.pvs

```diff
@@ -922,8 +995,10 @@ END cast_vote_records
 %%% the county-level, and one is for audit board members.
```

Comment:
Incorrect!! Audit Board members use the County interface. And should the public dashboard be mentioned here? Is it a "user interface"?